src/routes/categories/containers/CategoriesPage.js

```diff
@@ -117,7 +118,8 @@ export class CategoriesPage extends React.Component<PropsType> {
         {this.getContent(categoryModel)}
       </div>
     } else {
-      const error = new ContentNotFoundError({type: 'category', id: this.props.path, city: city, language})
+      const errorType = categoryPath.indexOf('extras') === 0 ? 'extra' : 'category'
```

Comment:
No, the route should just not end up in the categoriesPage in the first place. This is too hacky, sorry :D

Reply:
Also why does the Categories page get rendered when you try to load a extra? I think we should map all routes which contain a /extras/ to the extras route.

Reply:
@maxammann you changed the ExtrasRoute when adding the WohnraumboerseExtra to the WebApp. I'll change it back :+1: